ci/copy-unix-release-artifacts.sh

```diff
@@ -93,6 +93,8 @@ if [[ "$NAME" == "linux" ]]; then
     mkdir -p $OUTPUT_DIR/split-release/daml-libs/daml-script
     bazel build //daml-script/daml:daml-script-dars
     cp bazel-bin/daml-script/daml/*.dar $OUTPUT_DIR/split-release/daml-libs/daml-script/
+    bazel build //daml-script/daml3:daml3-script-dars
+    cp bazel-bin/daml-script/daml3/*.dar $OUTPUT_DIR/split-release/daml-libs/daml-script/
```

Comment:
Why do we need daml3 here ?

Reply:
We don't, that's a mistake, thanks for catching.